plugins/woocommerce/src/Admin/Features/OnboardingTasks/Tasks/Products.php

```diff
@@ -176,16 +158,7 @@ public function possibly_add_load_sample_return_notice_script( $hook ) {
 			return;
 		}
 
-		$script_assets_filename = WCAdminAssets::get_script_asset_filename( 'wp-admin-scripts', 'onboarding-product-notice' );
```

Comment:
I noticed the this has a different script name that is imported, `onboarding-product-notice` compared to the queued filename, `onboarding-load-sample-products-notice`.

I think by changing it to use the `register_script` function, it's now not imported anymore?

Reply:
It's still getting imported 😄 , and `onboarding-load-sample-products-notice` is the correct one because we just need to get its dependencies. 

It didn't cause the error with `onboarding-product-notice`  Because `onboarding-product-notice` dependencies are the superset of `onboarding-load-sample-products-notice`'s.